Examples/CaseStudies/InteractionWithAppStorageView.swift

```diff
@@ -57,22 +57,19 @@ struct InteractionWithAppStorageView: SwiftUICaseStudy {
   }
 }
 
-#Preview("Standard user defaults", traits: .dependency(\.defaultAppStorage, .standard)) {
+#Preview("Standard user defaults") {
+  let _ = prepareDependencies { $0.defaultAppStorage = .standard }
   NavigationStack {
     CaseStudyView {
       InteractionWithAppStorageView()
     }
   }
 }
 
-#Preview(
-  "Quarantined user defaults",
-  traits: .dependencies {
-    $0.defaultAppStorage = UserDefaults(
-      suiteName: "\(NSTemporaryDirectory())co.pointfree.Sharing.\(UUID().uuidString)"
-    )!
-  }
-) {
+#Preview("Quarantined user defaults") {
+  let _ = UserDefaults(
+    suiteName: "\(NSTemporaryDirectory())co.pointfree.Sharing.\(UUID().uuidString)"
+  )!
```

Comment:
Shouldn't it be wrapped in `prepareDependencies` and assigned to `defaultAppStorage`?